eras/alonzo/test-suite/test/Test/Cardano/Ledger/Alonzo/PlutusScriptExamples.hs

```diff
@@ -178,6 +179,7 @@ explainTest plutus mode ds =
         PlutusWithContext
           { pwcProtocolVersion = eraProtVerLow @era
           , pwcScript = Left plutus
+          , pwcScriptHash = mkScriptHash @(EraCrypto era) 1
```

Comment:
It seems that it might be okay to use a "dummy hash" here. Regardless, I thought that I will point this out and ask if this could cause some issues down the line?